mrdo_arcade.asm

```diff
@@ -8760,8 +8800,8 @@ old:
 	DB 005
 	DW BYTE_C294
 	DW MR_DO_PUSH_RIGHT_02_PAT
-	DB 000, 44*4, 000
-	DW FIVE_HUNDRED_SCORE_PAT
+;	DB 000, 44*4, 000

```

Comment:
If you keep my BONUS tiles loaded in the tileset, you can remove this from the sprite list